Resources/Textures/Structures/Specific/Atmospherics/gas_pipe_sensor.rsi/meta.json

```diff
@@ -0,0 +1,38 @@
+{
+    "version": 1,
+    "license": "CC0-1.0",
+    "copyright": "Created by chromiumboy (github) for SS14, based on the digital valve from /tg/",
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "states": [
+        {
+            "name": "icon"
+        },
+		{
+            "name": "base"
+        },
+		{
+            "name": "cavity"
+        },
+		{
+            "name": "wires"
+        },
+		{
+            "name": "wires_connected"
+        },
+		{
+            "name": "blank"
```

Comment:
It is a hack, because setting the layer visible or not in its generic visualizer wasn't having impact on the sprite visuals, but changing sprite state did for some weird reason.... I'll see if I can get it working without it

Reply:
Still doesn't respond to changes to visibility. Open to any suggestions :P

Reply:
OK, so the issue is because its has a subfloor component. This means that all layers are either set as being visible or not visible. So there is no alternative here, we need to use state changes